packages/electrode-archetype-react-app-dev/lib/dev-admin/dev-server.js

```diff
@@ -31,13 +39,18 @@ if (electrodeServer) {
       }
     },
     plugins: {
-      inert: {},
       webpackDevHapi: {
         module: "./dev-hapi.js",
         requireFromPath: __dirname
       }
     }
-  });
+  };
+  if (isInstalled("@hapi/inert")) {
```

Comment:
inert is no longer required because we have our own `replyFile` implementation now.